objectivec/GPBCodedInputStream.h

```diff
@@ -184,7 +184,7 @@ CF_EXTERN_C_END
  *                          extensions for message.
  **/
 - (void)readMessage:(GPBMessage *)message
-  extensionRegistry:(nullable GPBExtensionRegistry *)extensionRegistry;
+  extensionRegistry:(nullable NSObject<GPBExtensionRegistry> *)extensionRegistry;
```

Comment:
I'd love to have these as id as opposed to NSObject. Why do you need them as NSObject?

Reply:
I think I've had hiccups with that in the past, but will swap it, the change would be: `NSObject<GPBExtensionRegistry> *` → `id<GPBExtensionRegistry>` (i.e. - the `*` goes away also)

Reply:
Done